ykrt/src/frame/llvmbridge.rs

```diff
@@ -23,55 +23,11 @@ impl Module {
         Self(module)
     }
 
-    pub fn function(&self, name: *const i8) -> Function {
-        let func = unsafe { LLVMGetNamedFunction(self.0, name) };
-        debug_assert!(!func.is_null());
-        unsafe { Function::new(func) }
-    }
-
     pub fn datalayout(&self) -> LLVMTargetDataRef {
         unsafe { LLVMGetModuleDataLayout(self.0) }
     }
 }
 
-pub struct Function(LLVMValueRef);
```

Comment:
I didn't expect this to go. Just not used?

Reply:
This was only used to get access to the live AOT instructions. Since we now pass pointers, this is no longer needed.